### PR TITLE
feat: ttfb / latency example

### PR DESCRIPTION
**Pull request by piotrpcharactr**
## Describe your changes

[2.1.1] - 2023-09-18

- Added ttfb / latency test example

## Type of change

Please delete options that are not relevant.

- [x] Other non-breaking change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project and meets the industry quality standards
- [x] I have made corresponding changes to the documentation, `CHANGELOG.md` and `README.md`
- [x] I have updated the version in `package.json` according to [semantic versioning](https://semver.org/)
- [x] I have provided meaningful commit messages

### CODE REVIEW

File: examples/nodejs/latency.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const serverLatency = Date.now() - serverStart;
+  console.log(`serverLatency is ${serverLatency} ms`);
+
+  // get voice latencies
+  const voices = [20, 40, 151, 222];
+  for (const v of voices) {
+    let startTime = 0;
+    let timeToFirstByte = 0;
+    const stream = await sdk.tts.convertStreamDuplex(v, {
+      onData: (data) => {
```
Comment (wiktor-mazur) [inline examples/nodejs/latency.ts:23]: `data` is an unused variable. Please, run `yarn lint` and correct warnings/errors it prints out if possible.

File: examples/nodejs/latency.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            `voice "${v}" latency test failed: error code [${event.code}], msg: ${event.reason}`
+          );
+        }
+      },
+    });
+
+    startTime = Date.now();
+    stream.convert("Hello world from the charactr TTS Duplex Streaming.");
+    stream.close();
+    await sleep(2000);
```
Comment (wiktor-mazur) [inline examples/nodejs/latency.ts:48]: Is it not better to use simplex streaming? It's more used and we wouldn't have to "hack" by sleeping. Unless we want to test duplex specifically for some reason.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@charactr/api-sdk",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (wiktor-mazur) [inline package.json:3]: It's not a bugfix so I'd bump the middle number.

File: examples/nodejs/latency.ts (hunk 2 above)
Comment (piotrpcharactr) [reply examples/nodejs/latency.ts:48]: No. Duplex Streaming is latency optimized. Simplex streaming is throughput optimized. So this is the right one for this test.

I want this to work sequentially (to minimize any network congestion effects on the results). If there is another simple way to synchronize the execution of this loop I'm all ears.

**Review by amadeuszIgnacyk — APPROVED** [review]

**Pull request merged**